components/Table/Table.module.scss

```diff
@@ -1,5 +1,6 @@
 @import 'colors';
 @import 'fonts';
+@import 'layout';
```

Comment:
Try to put them in one line

`@import 'colors', 'fonts', 'layout';`

Reply:
Done